packages/cloudflare/src/cli/build/index.ts

```diff
@@ -276,3 +277,10 @@ export async function getLatestCompatDate(): Promise<string | undefined> {
     /* empty */
   }
 }
+
+function showErrorForOldNextjsVersions(options: buildHelper.BuildOptions) {
```

Comment:
this function name doesn't really convey the fact that this function exits the build process (by the name alone I would assume that this function simply logs the error)

I would maybe call it something like:
```
crushIfOnOldNextjsVersion
```
or something like that

If you where to throw as I suggested in my other comment it could also be:
```
warnAndThrowOnOldNextjsVersions
```
or something like that

Reply:
+1 for a better name, my take: `ensureNextjsVersionSupported`